git.md

```diff
@@ -0,0 +1,35 @@
+## git
+
+### commit
+
+`git commit` saves changes
+
+Example:
+```console
+$ git commit -m "Enter here a message"
+```
+### status
+`git status`  Shows the state of working directory.
+```console
+$ git status
+```
+### push
+`git push` It lets you move a local branch to another repository.
```

Comment:
Make it as a sentence: `git push` moves a local ...

Reply:
Done thank's